frontend-api/src/main/java/uk/gov/di/authentication/frontendapi/lambda/LoginHandler.java

```diff
@@ -164,14 +164,16 @@ public APIGatewayProxyResponseEvent handleRequestWithUserContext(
                         IpAddressHelper.extractIpAddress(input),
                         userProfile.getPhoneNumber(),
                         persistentSessionId,
-                        pair("internalSubjectId", userProfile.getSubjectID()));
+                        pair("internalSubjectId", userProfile.getSubjectID()),
+                        pair("attemptNoFailedAt", configurationService.getMaxPasswordRetries()));
 
                 return generateApiGatewayProxyErrorResponse(400, ErrorResponse.ERROR_1028);
             }
 
             if (!credentialsAreValid(request, userProfile)) {
                 codeStorageService.increaseIncorrectPasswordCount(request.getEmail());
-
+                var incorrectPasswordCountPair =
```

Comment:
You can do this, but is there any point not just declaring it inline on line 188? This seems to be the de facto standard approach across our code base so might read more easily when read alongside the existing code